src/main/java/com/sampleweb/pages/GroceryPage.java

```diff
@@ -0,0 +1,95 @@
+package com.sampleweb.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.util.List;
+
+public class GroceryPage extends HomePage {
+
+    public static String PATH = "https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Dgrocery&field-keywords=";
+    public static String HOMEPAGE_PATH = "https://www.amazon.co.uk/";
+
+    private String title = "Grocery";
+    private String pageTitle = "//h1[text()=\"Grocery\"]";
+    private String categorySearchDropdown = "//select[@id='searchDropdownBox']";
+    private String groceryCategorySearchDropdownOption = "//option[contains(text(),'Grocery')]";
+    private String globalStoreComponent = "//h4[contains(text(),'Global Store')]";
+    private String avgCustomerReviewComponent = "//h4[contains(text(),'Global Store')]";
+    private String submenuTitleComponent = "//span[@class='nav-a-content'][contains(text(),'Grocery')]";
+    private String sidemenuPrimeLogo = "//div[contains(@id, 'leftNav')]//i[contains(@class, 'a-icon-prime')]";
+    private String sidemenuPrimeLogoCheckbox = "../..//input[contains(@type, 'checkbox')]";
+    private String searchResults = "//div[contains(@class, 's-search-results')]//div[contains(@class, 's-result-item')]";
+    private String productPrimeLogo = "//i[contains(@class, 'a-icon-prime')]";
+    private String moreBuyingChoices = "//span[contains(text(),'More buying choices')]";
+
```

Comment:
Maybe some of these variables can be declared inside of the method that needs them? Who knows